packages/types/src/request-logic-types.ts

```diff
@@ -140,6 +140,33 @@ export interface IRequest {
   nonce?: number;
 }
 
+/** Pending request in request logic */
```

Comment:
I would call it "pending data of the request", "pending request" sounds like it's another request